catalog/core/catalog-core-standardframework/src/test/java/ddf/catalog/impl/CatalogFrameworkImplTest.java

```diff
@@ -1141,6 +1149,9 @@ public void testDelete() throws Exception {
                 1);
         when(mockFederationStrategy.federate(anyList(), anyObject())).thenReturn(queryResponse);
 
+        when(mockRemoteDeleteOperations.performRemoteDelete(any(), any())).then(returnsSecondArg());
+        deleteOperations.setRemoteDeleteOperations(mockRemoteDeleteOperations);
```

Comment:
Why not just set `deleteOperations.setRemoteDeleteOperations(mockRemoteDeleteOperations)` in the setup method? Then you would not have to duplicate this line for each of the tests